.eslintrc.js

```diff
@@ -22,24 +22,19 @@ const defaultRules = {
   'no-caller': 2,
   'no-console': 2,
   'no-else-return': 0,
-  'no-empty-character-class': 2,
```

Comment:
All (except `one-var`) removed rules are still applied because they're in `eslint:recommend` config. Just removing as duplicates.